### DISCUSSION TITLE
Learn TypeScript Project

### DISCUSSION THREAD

**Discussion by amandamartin-dev** [EddieHubCommunity/support#6315] [Ideas]
This is to begin a discussion started via this [Twitter post ](https://twitter.com/eddiejaoude/status/1608864720022290441) on Eddie's account

Idea: To increase contributions/overall interest in TS, can we create a project that would help folks learn by doing within EddieHub?

**Comment by YahefuYusufu** [comment]
i am interested

**Comment by benessilfie** [comment]
very much interested Amanda, is it in the works?

**Reply by amandamartin-dev** [reply]
Hello @benessilfie! I started this discussion thread based on a tweet @eddiejaoude posted about folks not contributing much if TS is the language. This area is to brainstorm how to perhaps create a way for beginners to learn while they contribute. The idea hasn't really been discussed yet, but please add your ideas! Eddie will likely want to weigh in on if this will be a good fit for the community.

Tl;Dr no not started yet and iso ideas for how it could work

**Comment by singhimat99** [comment]
I would love to be involved!

**Comment by money8203** [comment]
Don't know TS but would love to learn while contributing :)

**Comment by ThorOnTheRocks** [comment]
Yes I'm new to TypeScript but would love to contribute!

**Comment by YahefuYusufu** [comment]
i have some experience with typesctipt,i also ready to contribute

**Comment by neerajkumar161** [comment]
I have some experience in Typescript with NodeJs and Angular. I would love to contribute and learn from the community.

**Comment by Draxeytina** [comment]
It's a good idea. I've been learning typescript for a while but this would be very helpful on the side.

**Comment by ankur0904** [comment]
@amandamartin-dev I think there must be a structured path(different sections of projects) where the beginner can learn on the way while contributing to the different parts of the project.
For example Level 1 for the basic syntax of the Typescript etc.
